### ISSUE TITLE
Copper, brass and railway mixers don't work, work on "half" [Bug] 

### ISSUE THREAD

**Issue by v0vaku7ub** [iglee42/CreateCasing#68]
**Describe the bug**
Mixers from mod (besides industrial and creative) instead of 2x speed doing nothing.

**To Reproduce**
1. Launch Minecraft
2. Join a World
3. Place rixer
 3.1.1 Place railway mixer
 3.2.1 Try any recipe of item or potion
 3.1.2. Place copper mixer
 3.2.2. Try any potion recipe
 3.1.3. Place brass mixer
 3.2.3. Try any item recipe.
4. See, mixing animation
5. Realize, that nothing working and there is just cool animation.

**Expected behavior**
Railway and other mixer will be working 2 times faster

**Screenshots**
No screenshots, but there is video with demonstration of the bug https://youtu.be/xM-1s9TVuyM (Sorry for UI language and for accidental Steam)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - Minecraft Version: 1.19.2
 - Forge Version: 43.4.0
 - Mod Version 1.6.0
 - A lot of other stuff

**Additional context**
It was in singleplayer survival on my private curseforgw modpack

**Comment by iglee42** [comment]
I'm sorry I can't reproduce the bug, check if the recipe is really in the modpack

**Comment by iglee42** [comment]
Try to update the mod version to 1.6.0-ht1

**Comment by v0vaku7ub** [comment]
Still don't working

**Comment by iglee42** [comment]
Can you send me your logs please

**Comment by iglee42** [comment]
So can I have your logs when you use a mixer please

**Comment by judicius96** [comment]
Running Forge 43.3.9
Create 0.5.1.f
Encased 1.6.0-ht1
Alloyed 1.5a
JEI 11.6.0.1019

https://gist.github.com/judicius96/276ffc4ad507d62be3379671045a75a0

Train and Bronze Mixers cannot make steel. They lower and show animation but never complete and continuously mix. 
Copper and Industrial Iron Mixers work fine and complete the recipe,

While running the Train Mixer at 256 RPM, mixing is continuous with no output.
If I then break the gear, the gears in the block stop but the mixing particles and sound continue.
Individual Items still show in the Basin and then eventually become the steel that is to be crafted. 
Recipe only completes if gear is removed that is feeding RPM to the mixer.

Was able to duplicate behavior with the Brass Mixer, same as above.
Recipe doesn't seem to start when the mixer lowers and starts spinning but when the gear is removed and the spinning animation stops.


Hope that helps find the issue.

**Comment by iglee42** [comment]
There is no bronze mixer but thanks for the idea

**Comment by judicius96** [comment]
Sorry, Brass! I do that all the time lol

**Comment by parmstr2** [comment]
Running Forge 47.2.0
create-1.20.1-0.5.1.f
CreateCasing-1.20.1-1.6.0-fix1
jei-1.20.1-forge-15.3.0.4

I'm running into this issue with the brass and train mixers. From a little playing around with a creative motor I'm noticing that the issue only occurs when the RPM of the mixer is 32 RPM or below.  Like the normal mixer all of the other mixers start to function at 30 RPM. The train and brass mixers both start working at 33 RPM. I'm wondering if those mixers have somehow been set to a higher RPM threshold?

Edit 1: They also seem to stop working if I exceed 64 RPM.

Edit 2: It starts to work again if I exceed 128 RPM. Does not function between 64 to 128 (including 64 and 128) though.
